app/src/main/java/com/android/feedme/ui/LandingScreen.kt

```diff
@@ -67,7 +75,7 @@ fun LandingPage(navigationActions: NavigationActions) {
               tags = listOf("Meat"),
               time = 1.15,
               rating = 4.5,
-              userid = "PasDavid",
+              userid = "EtienneBoisson",
               difficulty = "Intermediate",
               "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.mamablip.com%2Fstorage%2FLasagna%2520with%2520Meat%2520and%2520Tomato%2520Sauce_3481612355355.jpg&f=1&nofb=1&ipt=8e887ba99ce20a85fb867dabbe0206c1146ebf2f13548b5653a2778e3ea18c54&ipo=images"),
```

Comment:
```suggestion
              "https://example.com"),
```
This is a very long URL that can just be an example URL.

(Also, why are testing things mixed in with real code?)